### PR TITLE
100% translated da.json. README.md with da added

### PR DESCRIPTION
**Pull request by henrikfalk**
First chunk of lines.

### CODE REVIEW

File: da.json [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-			"option-line-wrap-description": "Wrap long lines to fit within the window, without having to scroll from left to right.",
-			"option-auto-convert-html": "Auto convert HTML",
-			"option-auto-convert-html-description": "Automatically convert HTML to Markdown when pasting and drag-and-drop from webpages. Use Ctrl/Cmd+Shift+V to paste without conversion.",
-			"option-vim-key-bindings": "Vim key bindings",
-			"option-vim-key-bindings-description": "Let you use Vim key bindings in the source editor."
+			"option-default-view-mode-preview": "Forhåndsvisning",
+			"option-show-line-number": "Vis linjenummer",
+			"option-show-line-number-description": "Vis linjenummer i margen.",
+			"option-use-tabs": "Benyt tabulator",
+			"option-use-tabs-description": "Benyt tabulator til at indrykke ved at trykke på "Tab" tasten. Slå den fra for at benytte mellemrumstasten.",
```
Comment (ericaxu) [inline da.json:38]: Please escape the quotes by using `\"` instead of just `"`, since the quote character would end the string.

**Comment by ericaxu** [comment]
Looks good! I added a comment.

File: da.json (hunk 1 above)
Comment (henrikfalk) [reply da.json]: I have corrected it. Seems to add a second commit in this pull request. Hope this is correct.

**Comment by ericaxu** [comment]
Great work! Let me know when it's ready to be merged.

**Comment by henrikfalk** [comment]
I have finished the translation of the language file.
It's ready to be merged.
When included in Obsidian I will test and correct any typos etc.

**Comment by ericaxu** [comment]
Looks good! I'll mark `da` as a completed language asap. Thanks for the amazing work!

**Pull request merged**